vnext/JSI/Shared/ChakraRuntime.cpp

```diff
@@ -613,6 +666,28 @@ bool ChakraRuntime::instanceOf(const facebook::jsi::Object &obj, const facebook:
 
 #pragma endregion Functions_inherited_from_Runtime
 
+// Rewrite some error messages
+void ChakraRuntime::RewriteErrorMessage(JsValueRef jsError) {
+  JsPropertyIdRef messagePropertyId;
+  VerifyChakraErrorElseThrow(JsGetPropertyIdFromName(L"message", &messagePropertyId));
+  JsValueRef message;
+  VerifyChakraErrorElseThrow(JsGetProperty(jsError, messagePropertyId, &message));
+  JsValueType valueType;
+  VerifyChakraErrorElseThrow(JsGetValueType(message, &valueType));
+  if (valueType == JsValueType::JsString) {
+    wchar_t const *messageStr;
+    size_t messageLength;
+    VerifyChakraErrorElseThrow(JsStringToPointer(message, &messageStr, &messageLength));
+    // JSI unit tests expect V8 or JSC like message on stack overflow.
+    if (std::wstring_view{L"Out of stack space"} == std::wstring_view{messageStr, messageLength}) {
```

Comment:
>L" [](start = 26, length = 2)

aren't these localized? (in C# I remember exception errors were localized)

Reply:
Maybe. Unfortunately, the JSI unit tests look for a specific text in the exception and I have no other way to pass the JSI tests without "cheating" here. Unit tests do not use localized strings. So, we should be good here.

---
In reply to: [460193326](https://github.com/microsoft/react-native-windows/pull/5461#discussion_r460193326) [](ancestors = 460193326)